ulohy_07_hangman/ai.py

```diff
@@ -0,0 +1,25 @@
+from random import choice
+
+
+def load_words():
+    """Vytvoří seznam slov ze souboru"""
+    filename = "words.txt"
+    try:
+        with open(filename, encoding="utf-8") as f:
+            contents = f.read()
+            words = contents.split("\n")
+            return words
+    except OSError:
+        print(f'Soubor {filename} bohužel nelze nalézt.')
+
+
+def choose_word(words):
+    """Vybere náhodného slovo ze seznamu"""
+    word = choice(words)
+    return word
```

Comment:
Pěkné použití  `choice()`.

V takhle jednoduché funkci, klidně rovnou `return choice(words)`. Proměnná nepomáhá čitelností ani se s ní nikde jinde nepracuje, je tam zbytečně navíc. Smysl by to dávalo, kdyby se např. vybrané slovo mělo někam logovat nebo printovat, to se zde neděje.

Reply:
Opraveno.